test/test_utils.py

```diff
@@ -251,12 +251,15 @@ def __init__(self):
 
 
 @contextmanager
-def server_for_test_app(test_app):
+def server_for_test_app(test_app, host=None, port=None):
     app = test_app.app
-    create_kwds = {
-    }
-    if os.environ.get("PULSAR_TEST_FILE_SERVER_HOST"):
-        create_kwds["host"] = os.environ.get("PULSAR_TEST_FILE_SERVER_HOST")
+
+    create_kwds = {}
+    if host:
+        create_kwds["host"] = host
+    if port:
+        create_kwds["port"] = port
+
     server = StopableWSGIServer.create(app, **create_kwds)
```

Comment:
For reviewers: now the Pulsar app won't bind to `PULSAR_TEST_FILE_SERVER_HOST` (only the files server). Please consider if this breaks anything before approving the changes.